test/D/SharedObjects/Common/common.py

```diff
@@ -45,8 +45,10 @@ def testForTool(tool):
         test.skip_test("Required executable for tool '{0}' not found, skipping test.\n".format(tool))
 
     if tool == 'gdc':
-        result = check_output(('gdc', '--version'))
-        version = result.decode().splitlines()[0].split()[3]
+        cp = subprocess.run(('gdc', '--version'), capture_output=True)
+        # gdc (GCC) 11.1.1 20210531 (Red Hat 11.1.1-3)\nCopyright (C)...
```

Comment:
What did the older versions output look like? will this break those?

Reply:
I don't have anything really old, but this is a really standard layout for the version message for the Gnu Compiler Collection: `tool (GCC) x.y.x reldate buiildinfo` so I'd be surprised if there's been any change for a very long time.

Reply:
The gcc tool uses something also head-scratchy, and of course different:
```
    # --version variant:
    match = re.search(r'[0-9]+(\.[0-9]+)+', line)
    if match:
        version = match.group(0)
```

Dunno if that's better or worse